test/views/other/gambling/mgd/register/RegisterMGDViewSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.other.gambling.mgd.register
+
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.other.gambling.mgd.register.registerMGD
+
+class RegisterMGDViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "registerMGD"
+
+  def createView = () => registerMGD(frontendAppConfig)(HtmlFormat.empty)(fakeRequest, messages)
+
+  "RegisterMGD view" must {
+    behave like normalPage(createView, messageKeyPrefix)
+
+    "Render the correct content" in {
+      val doc =  asDocument(createView())
+      val view = doc.text()
```

Comment:
There are no tests for the additional paragraphs on the page, we should always be following TDD and writing tests